.github/workflows/notify-doc.yml

```diff
@@ -0,0 +1,23 @@
+name: notifier for docteam
+
+on:
+    pull_request_target:
+        types: [ labeled, closed ]
+
+jobs:
+    build:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout repo content
+              uses: actions/checkout@v2
+
+            - name: Setup python
+              uses: actions/setup-python@v2
+              with:
+                  python-version: 3.9
+
+            - name: Install python packages 2
+              run: pip install PyGithub
```

Comment:
Let's better add all necessary libs into `requirements.txt` and install it via ``
It allows not to change workflow on each new library

see https://github.com/intellij-rust/intellij-rust/blob/master/.github/workflows/create-changelog.yml#L26-L27 as example

Reply:
Having `requirements.txt` and `Pipfile` in the same directory looks confusing - you are mixing two ways of dependency management. As an improvement idea for a future PR, I would suggest either

- getting rid of `Pipfile` and having two `requirements.txt`, e.g. `requirements-cli.txt` and `requirements-actions.txt` if you would like to have the dependencies split
- or having all requirements in a single `Pipfile` (if you like `Pipenv` 🙂), you can have CLI requirements as [dev-packages](https://pipenv-fork.readthedocs.io/en/latest/basics.html#example-pipfile) there to separate them from actions requirements

Reply:
I think we will drop `Pipfile`. It was introduced just to try something more interesting than `requirements.txt` but looks like we actually don't need it